datasets/1.0/huggingface-anthropic-hh-rlhf/metadata.json

```diff
@@ -266,6 +303,31 @@
       "@id": "harmless-base",
       "name": "harmless-base",
       "field": [
+        {
+          "@type": "cr:Field",
+          "@id": "harmless-base/split",
```

Comment:
Are the fields repeated between the RecordSets?

Reply:
Exactly -- is there a better way to express this with less redundancy?

Reply:
We could parametrize the config name. But that'd be a feature in itself, so no need to do this in this PR. Good to know we have a dataset to develop this feature!